Gifski/DraggableFile.swift

```diff
@@ -13,8 +13,17 @@ final class DraggableFile: NSImageView {
 		super.init(frame: frame)
 		isEditable = false
 		unregisterDraggedTypes()
+		
+		superview?.wantsLayer = true
```

Comment:
Subviews should not set `wantsLayer` of its superview. This statement should be in the superview instead.